demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -0,0 +1,406 @@
+                    "type": "python",
+                    "request": "launch",
+                    "program": str(integration_script.path.with_suffix(".py")),
+                    "console": "integratedTerminal",
+                    "cwd": "${workspaceFolder}",
+                    "justMyCode": False,
+                },
+                {
+                    "name": "Python: Debug Tests",
+                    "type": "python",
+                    "request": "launch",
+                    "program": "${file}",
+                    "purpose": ["debug-test"],
+                    "console": "integratedTerminal",
+                    "justMyCode": False,
+                },
+            ],
+        }
+        with open(ide_folder / "launch.json", "w") as f:
+            json.dump(launch, f, indent=4)
+
+
+def configure_vscode(
+    ide_folder: Path,
+    integration_script: IntegrationScript,
+    test_docker_image: str,
+    interpreter_path: Path,
+):
+    """This functions configures VSCode for the integration script
+
+    Args:
+        ide_folder (Path): The ".vscode" folder to configure
+        integration_script (IntegrationScript): The integration script to configure
+        test_docker_image (str): The test docker image to use for running tests
+        interpreter_path (Path): The local interpreter path to configure
+    """
+    ide_folder.mkdir(exist_ok=True)
+    configure_vscode_settings(ide_folder, integration_script, interpreter_path)
+    configure_vscode_tasks(ide_folder, integration_script, test_docker_image)
+    configure_vscode_launch(ide_folder, integration_script)
+
+
+def install_virtualenv(
+    integration_script: IntegrationScript,
+    test_docker_image: str,
+    overwrite_virtualenv: bool,
+) -> Path:
+    docker_client = docker_helper.init_global_docker_client()
+    requirements = (
+        docker_client.containers.run(
+            test_docker_image, command="pip list --format=freeze", remove=True
+        )
+        .decode()
+        .split("\n")
+    )
+    venv_path = integration_script.path.parent / "venv"
+    interpreter_path = venv_path / "bin" / "python"
+    if venv_path.exists() and not overwrite_virtualenv:
+        return interpreter_path
+    logger.info(f"Creating virtualenv for {integration_script.name}")
+    shutil.rmtree(venv_path, ignore_errors=True)
+    venv.create(venv_path, with_pip=True)
+    for req in requirements:
+        try:
+            if not req:
+                continue
+            subprocess.run(
+                [
+                    f"{venv_path / 'bin' / 'pip'}",
+                    "-q",
+                    "--disable-pip-version-check",
+                    "install",
+                    req,
+                ],
+                check=True,
+            )
+            logger.info(f"Installed {req}")
+        except subprocess.CalledProcessError:
+            logger.warning(f"Could not install {req}, skipping...")
```

Comment:
Does skipping make sense? Don't we want to just give up if such installation fails?

Reply:
yes, because the integration might be able to run/debug without all the dependencies. However, we print a warning so the use will be aware (and try to fix/run in docker)